scripts/plugin_runner/create_venv.py

```diff
@@ -0,0 +1,313 @@
+"""
+This script allows you to create a virtual environment
+and install the requirements gathered from a given dataset with python projects.
+
+It accepts
+    * path to the folder with python projects.
+    * path to the folder where you want to create the environment.
```

Comment:
Please, add some description what happens if the one library has different versions in the different projects